app/controllers/api/v1/visits_controller.rb

```diff
@@ -6,7 +6,32 @@ class VisitsController < ApplicationController
       attr_accessor :visit
 
       def index
```

Comment:
Please refactor this action logic, it's seems not good variant of implementation. Think about that